gen_examples.py

```diff
@@ -245,26 +308,69 @@ def example1(draft):
                        bundle_policy = 'balanced', mux_policy = 'negotiate',
                        local_ip = '192.0.2.2', stun_ip = None, relay_ip = None,
                        fingerprint = fp2, m_sections = ms2)
+
   o = pc1.create_offer()
   output_desc('offer-A1', o, draft)
   a = pc2.create_answer()
   output_desc('answer-A1', a, draft)
 
-def example2():
-  pc1 = PeerConnection(trickle = True, bundle_policy = 'max-bundle',
-                       mux_policy = 'require')
-  pc2 = PeerConnection(trickle = False, bundle_policy = 'max-bundle',
-                       mux_policy = 'require')
+def example2(draft):
+  ms1 = [
+    { 'type': 'audio', 'mid': 'a1',
+      'ms': '57017fee-b6c1-4162-929c-a25110252400',
+      'mst': 'e83006c5-a0ff-4e0a-9ed9-d3e6747be7d9',
+      'local_port': 51556, 'srflx_port': 52546, 'relay_port': 61405,
+      'ice_ufrag': 'ATEn', 'ice_pwd': 'AtSK0WpNtpUjkY4+86js7ZQl',
+      'dtls_dir': 'passive' },
+    { 'type': 'application', 'mid': 'd1' }
+  ]
+  fp1 = '29:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04:BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2'
+  pc1 = PeerConnection(session_id = '4962303333179871723', trickle = True,
+                       bundle_policy = 'max-bundle', mux_policy = 'require',
+                       local_ip = '192.168.1.2', stun_ip = '11.22.33.44',
+                       relay_ip = '22.33.44.55',
+                       fingerprint = fp1, m_sections = ms1)
+
+  ms2 = [
+    { 'type': 'audio', 'mid': 'a1',
+      'ms': '71317484-2ed4-49d7-9eb7-1414322a7aae',
+      'mst': '6a7b57b8-f043-4bd1-a45d-09d4dfa31226',
+      'local_port': 61665, 'srflx_port': 64532, 'relay_port': 50416,
```

Comment:
I think it would be nice if the ports for each of the addresses  had the last 3 digits of the port be the same as the last tuple of the IP.